phpBB/phpbb/db/migration/data/v310/style_update_p1.php

```diff
@@ -69,10 +69,11 @@ public function styles_update()
 		$skip_dirs = array('.', '..', 'prosilver');
 		foreach ($iterator as $fileinfo)
 		{
-			if ($fileinfo->isDir() && !in_array($fileinfo->getFilename(), $skip_dirs) && file_exists($fileinfo->getPathname() . '/style.cfg'))
```

Comment:
Should probably be rewritten to both handle pre 3.3 and 3.3 styles, so both cfg and json.